### PR TITLE
[Yangjie] Add component product detail.

### PR DESCRIPTION
**Pull request by JellyLu**

### CODE REVIEW

File: src/components/ProductDetail/ProductDetail.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.product-img {
+  height: 200px;
+  width: 100%;
+}
+
+.product-brief {
+  display: flex;
+  font-size: 16pt;
```
Comment (liujiefengfeng) [inline src/components/ProductDetail/ProductDetail.css]: maybe px ?

Comment (liujiefengfeng) [inline src/components/ProductDetail/ProductDetail.css]: maybe px ?

File: src/components/ProductDetail/ProductDetail.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.ProdcutDetail {
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+.product-img {
+  height: 200px;
```
Comment (liujiefengfeng) [inline src/components/ProductDetail/ProductDetail.css:7]: it is better to keep the height and width both px or both percent.

File: src/components/ProductDetail/ProductDetail.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.product-details {
+  padding-right: 10%;
+  padding-left: 10%;
+  margin-top: 20px;
+  margin-bottom: 50px;
+  text-align: left;
+}
+
+.small {
```
Comment (liujiefengfeng) [inline src/components/ProductDetail/ProductDetail.css]: it looks that ProductDetail component never use this css class.

File: src/components/ProductDetail/ProductDetail.css (hunk 2 above)
Comment (JellyLu) [inline src/components/ProductDetail/ProductDetail.css]: done

**Pull request merged**